docs/src/examples/components/Input/Variations/InputExampleWrapperSlot.shorthand.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import { Grid, Input, Text } from '@stardust-ui/react'
+
+const InputExampleWrapperSlot = () => (
+  <Grid columns="1fr 2fr" styles={{ justifyItems: 'start', alignItems: 'center', gap: '10px' }}>
+    <Text content="Input default:" />
+    <Input
+      placeholder="Search..."
+      dir="ltr"
+      tabIndex={2}
+      styles={{ color: 'blue', backgroundColor: 'yellow' }}
+    />
+
+    <Text content="Wrapped Input with props:" />
+    <Input
+      placeholder="Search..."
+      dir="ltr"
+      tabIndex={2}
+      styles={{ color: 'blue', backgroundColor: 'yellow' }}
+      wrapper={{
+        // override component's 'dir' attribute
```

Comment:
nit: just to explain from the _result_'s point of view to the client, not the _intent_ - maybe we might consider to use `will override component's 'dir' attribute` form for this comment instead